selenium/che-selenium-core/bin/webdriver.sh

```diff
@@ -412,13 +425,13 @@ HOW TO of usage:
 
     Rerun failed tests:
         ${CALLER} <...> --failed-tests
-        ${CALLER} <...> --failed-tests --rerun
+        ${CALLER} <...> --failed-tests --rerun [ATTEMPTS]
 
     Debug selenium test:
         ${CALLER} -Mlocal --test=<TEST> --debug
 
     Analyse tests results:
-        ${CALLER} --compare-with-ci [CI job number]
+        ${CALLER} --compare-with-ci [BUILD NUMBER]
```

Comment:
`BUILD NUMBER` collocation usually references to the product build number but `CI job number` indicates the number of the job on the CI.

Reply:
You might be right. From this point, `[JOB BUILD]` sounds better, doesn't it?

Reply:
ok

Reply:
Actually it is a job number.

Reply:
Job doesn't have a number. It has a _Project Name_ which job is aimed to execute https://wiki.jenkins.io/display/JENKINS/Building+a+software+project. And each execution is named _build_ and has certain number. For example, URL with interesting test report
this link https://ci.codenvycorp.com/view/qa/job/che-integration-tests-che6/100/testReport/ has:
- **che-integration-tests-che6**: name of project which job executes; 
- **100**: number of execution of project, that is the build number. 

Therefore, we could name this number **100** as a `[PROJECT BUILD NUMBER]` and it can be confused with Che build namber, or as `[JOB BUILD NUMBER]` which is correct, but IMHO can be shorten to `[JOB BUILD]`.

Reply:
ok, `BUILD_NUMBER` makes sense then.